README.rst

```diff
@@ -121,6 +121,13 @@ these exact identifiers directly, if you need something specific.
 The ``source`` release is always automatically included. ``pip`` will use
 this as a fallback in the case a suitable wheel cannot be found.
 
+Requirements can use PEP-0496 style specifiers (e.g. like
```

Comment:
Please change that to `[PEP-0496](https://www.python.org/dev/peps/pep-0496/)` instead of `PEP-0496`.